src/connect/components/sidebar/sidebar-user.tsx

```diff
@@ -24,9 +24,7 @@ export const SidebarUser: React.FC<SidebarUserProps> = ({
         />
         <div className="collapsed:hidden expanding:hidden">
             <p className="text-sm font-semibold text-gray-800">{username}</p>
-            <p className="text-text-gray-600 text-xs font-semibold">
```

Comment:
this was actually a bug that it caught — by accidentally doing `text-text-` it made the class not actually get applied